### PR TITLE
Add the 'ping' group to bikeshed.

### PR DESCRIPTION
**Pull request by jyasskin**
@wseltzer @pes @tjwhalen @sandandsnow This will control how Bikeshed generates PING drafts, initially https://w3cping.github.io/privacy-threat-model.

Are there any other pieces of boilerplate I should include here? You can find examples in https://github.com/tabatkins/bikeshed/tree/master/bikeshed/boilerplate/webappsec.

### CODE REVIEW

**Comment by tabatkins** [comment]
Typically you're gonna want customized status, copyright, and maybe header too; check out other w3c boilerplate folders.

**Comment by jyasskin** [comment]
Ok, here's a commit with that boilerplate. I have no strong opinions about the actual content.

File: bikeshed/boilerplate/ping/copyright.include [hunk 1]
```diff
@@ -0,0 +1 @@
+ <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> © 2019 <a href="https://www.w3.org/"><abbr title="World Wide Web Consortium">W3C</abbr></a><sup>®</sup> (<a href="https://www.csail.mit.edu/"><abbr title="Massachusetts Institute of Technology">MIT</abbr></a>, <a href="https://www.ercim.eu/"><abbr title="European Research Consortium for Informatics and Mathematics">ERCIM</abbr></a>, <a href="https://www.keio.ac.jp/">Keio</a>, <a href="https://ev.buaa.edu.cn/">Beihang</a>). <abbr title="World Wide Web Consortium">W3C</abbr> <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Legal_Disclaimer">liability</a>, <a href="https://www.w3.org/Consortium/Legal/ipr-notice#W3C_Trademarks">trademark</a> and <a rel="license" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document">permissive document license</a> rules apply.
```
Comment (hober) [inline bikeshed/boilerplate/ping/copyright.include]: Probably shouldn't hardcode 2019 here

File: bikeshed/boilerplate/ping/status-ED.include [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <abbr title="World Wide Web Consortium">W3C</abbr> publications and the
+    latest revision of this technical report can be found in the <a
+    href="https://www.w3.org/TR/"><abbr title="World Wide Web
+    Consortium">W3C</abbr> technical reports index</a>.</em>
+  </p>
+
+  <p>
+    This document was published by the
+    <a href="https://www.w3.org/2019/privacy/">Privacy Interest Group</a> as an
+    [LONGSTATUS]. This document is intended to become a W3C Recommendation.
```
Comment (hober) [inline bikeshed/boilerplate/ping/status-ED.include]: Interest Groups can't pubish Recommendations.

File: bikeshed/boilerplate/ping/copyright.include (hunk 1 above)
Comment (jyasskin) [reply bikeshed/boilerplate/ping/copyright.include]: Indeed, thanks: #1574

File: bikeshed/boilerplate/ping/status-ED.include (hunk 2 above)
Comment (jyasskin) [reply bikeshed/boilerplate/ping/status-ED.include]: Oops, thanks: #1575

**Pull request merged**